src/overlays/actors/ovl_Toudai/ac_toudai.c

```diff
@@ -1,51 +1,287 @@
+#include "global.h"
 #include "ac_toudai.h"
 #include "overlays/gamestates/ovl_play/m_play.h"
+#include "c_keyframe.h"
+#include "m_field_info.h"
+#include "m_collision_bg.h"
+#include "m_rcp.h"
+#include "m_lib.h"
+#include "overlays/actors/ovl_Structure/ac_structure.h"
+
+#define THIS ((Toudai*)thisx)
 
 void aTOU_actor_ct(Actor* thisx, Game_Play* game_play);
 void aTOU_actor_dt(Actor* thisx, Game_Play* game_play);
 void aTOU_actor_init(Actor* thisx, Game_Play* game_play);
 void aTOU_actor_draw(Actor* thisx, Game_Play* game_play);
 
-#if 0
 ActorProfile Toudai_Profile = {
-    /* */ ACTOR_TOUDAI,
+    /* */ 175, // ACTOR_TOUDAI
     /* */ ACTOR_PART_0,
     /* */ 0,
     /* */ 0x5843,
-    /* */ GAMEPLAY_KEEP,
+    /* */ 3, // GAMEPLAY_KEEP
```

Comment:
What's wrong with using the enum values?

Reply:
I wasn't sure if there's some header I need to include to use them, or if I need to make an enum somewhere

Reply:
Oh right, I forgot to include the files to use those enums. You want `m_actor_dlftbls.h` and `m_object.h`